CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# v0.3.4 (unreleased)
```

Comment:
```suggestion
# v0.3.4 (2020-05-27)
```

Or 5/28.

I think it's better to have a chance of this (not very important) date being wrong rather than having the commit in the image names be wrong.